server/models/Availability.js

```diff
@@ -0,0 +1,97 @@
+const mongoose = require("mongoose");
+
+const availabilitySchema = new mongoose.Schema({
+  profileId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'Profile'
+  },
+  schedule: {
+    'monday' : {
+      startTime: {
+        type: Date
+      }, 
+      endTime: {
+        type: Date
+      },
+      active: {
+        type: Boolean,
+        default: false 
+      }
```

Comment:
We can refactor this into a schema of its own to avoid repeating ourselves (see [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself#:~:text=%22Don't%20repeat%20yourself%22,data%20normalization%20to%20avoid%20redundancy.))